ansible/tasks/initdb.yml

```diff
@@ -61,4 +92,17 @@
     password: "{{ db_password }}"
     force_basic_auth: yes
   with_items: "{{ db.authkeys }}"
-  when: result.status == 404
+  when: dbexists is defined and dbexists.status == 404
+
+- name: secure db against anonymous access
```

Comment:
hmm is there a way to put this in the couchdb role? this feels like it is in the wrong place here.